src/alfasim_score/constants.py

```diff
@@ -46,3 +46,10 @@
 GAS_LIFT_VALVE_DEFAULT_DIAMETER = Scalar(0.25, DIAMETER_UNIT, "diameter")
 GAS_LIFT_VALVE_DEFAULT_DISCHARGE = Scalar(0.826, FRACTION_UNIT)
 GAS_LIFT_VALVE_DEFAULT_DELTA_P_MIN = Scalar(0.0, PRESSURE_UNIT)
+
+# numerical and time options default values
+MAXIMUM_TIMESTEP_CHANGE_FACTOR = 1.2
+NUMERICAL_TOLERANCE = 1.0e-3
+INITIAL_TIMESTEP = Scalar(0.1, TIME_UNIT)
```

Comment:
```
INITIAL_TIMESTEP = Scalar(0.1, TIME_UNIT)
MINIMUM_TIMESTEP = Scalar(1.0e-4, TIME_UNIT)
MAXIMUM_TIMESTEP = Scalar(2.0, TIME_UNIT)
```
These values are supposed to be in seconds, but `TIME_UNIT` is minutes.

Reply:
Fixed.